### ISSUE TITLE
Invalid CTPPSDetId error in 11624.911

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#32653]
Workflow 11624.911 step 1 failed in CMSSW_11_3_X_2021-01-12-2300 and in CMSSW_11_3_X_2021-01-12-1100 with an exception
```
Begin processing the 8th record. Run 1, Event 8, LumiSection 1 on stream 0 at 13-Jan-2021 04:40:14.281 CET
----- Begin Fatal Exception 13-Jan-2021 04:40:24 CET-----------------------
An exception of category 'InvalidDetId' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 5 stream: 3
   [1] Running path 'FEVTDEBUGoutput_step'
   [2] Prefetching for module PoolOutputModule/'FEVTDEBUGoutput'
   [3] Calling method for module OscarMTProducer/'g4SimHits'
Exception Message:
CTPPSDetId ctor: Invalid parameters: arm=4294967295 station=4294967295 rp=4294967295
----- End Fatal Exception -------------------------------------------------
```

**Comment by makortel** [comment]
assign simulation, geometry

**Comment by cvuosalo** [comment]
I have reproduced the error and am investigating it further.

**Comment by cvuosalo** [comment]
@ianna This error is due to the PPS source code searching for a physical volume with name "RP_box_primary_vacuum" while the DD4hep name is "RP_Box_023:RP_box_primary_vacuum_10023". What is the best way to perform a search for a physical volume name that will work for both old DD and DD4hep?

**Comment by cvuosalo** [comment]
Fixed in PR #32667.

**Comment by cvuosalo** [comment]
+1
